crates/bevy_ecs/src/query/state.rs

```diff
@@ -95,6 +95,26 @@ impl<D: QueryData, F: QueryFilter> QueryState<D, F> {
     ) -> &QueryState<NewD, NewF> {
         &*(self as *const QueryState<D, F> as *const QueryState<NewD, NewF>)
     }
+
+    /// Returns the archetype components accessed by this query.
```

Comment:
```suggestion
    /// Returns the archetype-components accessed by this query.
```

Reply:
Hmm, "archetype component" is a compound noun which I feel usually reads better without a hyphen. I prefer to save hyphens for compound adjectives